libs/styles/styles.css

```diff
@@ -453,6 +459,38 @@ span.icon.margin-inline-end { margin-inline-end: 8px; }
 
 .con-button.button-xl span.icon.margin-left { margin-left: 14px; }
 
+/* rounded border styles*/
+.s-rounded-corners,
+.s-rounded-corners > .background img {
+  border-radius: var(--s-rounded-corners);
+}
+
+.m-rounded-corners,
+.m-rounded-corners > .background img {
+  border-radius: var(--m-rounded-corners);
+}
+
+.l-rounded-corners,
+.l-rounded-corners > .background img {
+  border-radius: var(--l-rounded-corners);
+}
+
+.s-image-rounded-corners .foreground picture img {
+  border-radius: var(--s-rounded-corners);
+}
+
+.m-image-rounded-corners .foreground picture img {
+  border-radius: var(--m-rounded-corners);
+}
+
+.l-image-rounded-corners .foreground picture img {
+  border-radius: var(--l-rounded-corners);
+}
```

Comment:
For brevity purposes, since this is the main style file and we want to keep it small, could we not group these with their non-image counterparts? As in
```
.s-rounded-corners,
.s-rounded-corners > .background img,
.s-image-rounded-corners .foreground picture img {
  border-radius: var(--s-rounded-corners);
}
```

Reply:
+1 this.
I think grouping like this would also help if/when an update is needed.